openpower/package/hostboot/p9Patches/hostboot-1000-fix-reboot-hang-when-data-in-hbel.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/src/usr/errl/errlmanager_common.C b/src/usr/errl/errlmanager_common.C
```

Comment:
Can't pick up patches that do not have Gerrit Change-IDs in them because it breaks the auto-update tool the Hostboot team has.